straxen/plugins/bayes_peak_classification.py

```diff
@@ -86,30 +85,40 @@ def _compute_wf_and_quantiles(data, sample_length, bayes_n_nodes: int):
     step_size = int(num_samples/bayes_n_nodes)
     steps = np.arange(0, num_samples+1, step_size)
     inter_points = np.linspace(0., 1.-(1./bayes_n_nodes), bayes_n_nodes)
-    
+    cumsum_steps = np.zeros(bayes_n_nodes + 1, dtype=np.float64)
+    frac_of_cumsum = np.zeros(num_samples + 1)
+    sample_number_div_dt = np.arange(0, num_samples+1, 1)
     for i, (waveform, dt) in enumerate(zip(data, sample_length)):
```

Comment:
I just confused myself here :D. `waveform` is a thing from `data` (len  == 200) and not `waveforms` (len==50). Maybe we should change the naming.

Reply:
sure, will change it to `samples`